src/intervals/functions.jl

```diff
@@ -4,61 +4,140 @@
 # Integer power:
 function ^{T<:Real}(a::Interval{T}, n::Integer)
```

Comment:
Some of the ITF1788 are really repetitive, but others are very sharp; maybe we could select few of them only...